File: trunk/serviceConsultaReniec/src/main/java/reniec/Persona.java
package reniec;

public class Persona {
	
	private String dni;
	private String nombre;
	private String apellidoPaterno;
	private String apellidoMaterno;
	private String fechaNacimiento;
	private String direccion;
	private String estadoCivil;
	
	public String getDni() {
		return dni;
	}

	public void setDni(String dni) {
		this.dni = dni;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getApellidoPaterno() {
		return apellidoPaterno;
	}

	public void setApellidoPaterno(String apellidoPaterno) {
		this.apellidoPaterno = apellidoPaterno;
	}

	public String getApellidoMaterno() {
		return apellidoMaterno;
	}

	public void setApellidoMaterno(String apellidoMaterno) {
		this.apellidoMaterno = apellidoMaterno;
	}

	public String getFechaNacimiento() {
		return fechaNacimiento;
	}

	public void setFechaNacimiento(String fechaNacimiento) {
		this.fechaNacimiento = fechaNacimiento;
	}

	public String getDireccion() {
		return direccion;
	}

	public void setDireccion(String direccion) {
		this.direccion = direccion;
	}

	public String getEstadoCivil() {
		return estadoCivil;
	}

	public void setEstadoCivil(String estadoCivil) {
		this.estadoCivil = estadoCivil;
	}

	public Persona(String dniTmp, String nombreTmp, String apePat, String apeMat, String fecNac, String direc, String estado) {
		this.dni=dniTmp;
		this.nombre= nombreTmp;
		this.apellidoPaterno=apePat;
		this.apellidoMaterno=apeMat;
		this.fechaNacimiento=fecNac;
		this.direccion=direc;
		this.estadoCivil=estado;
	}
	
}

File: Fuentes/Aplicativo/Admision/Models/Persona.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Globalization;
using System.Web.Mvc;
using System.Web.Security;

namespace Admision.Models
{

    public class Persona
    {
        private String dni;

        public String Dni
        {
            get { return dni; }
            set { dni = value; }
        }
        private String nombre;

        public String Nombre
        {
            get { return nombre; }
            set { nombre = value; }
        }
        private String apellidoPaterno;

        public String ApellidoPaterno
        {
            get { return apellidoPaterno; }
            set { apellidoPaterno = value; }
        }
        private String apellidoMaterno;

        public String ApellidoMaterno
        {
            get { return apellidoMaterno; }
            set { apellidoMaterno = value; }
        }
        private String fechaNacimiento;

        public String FechaNacimiento
        {
            get { return fechaNacimiento; }
            set { fechaNacimiento = value; }
        }
        private String direccion;

        public String Direccion
        {
            get { return direccion; }
            set { direccion = value; }
        }
        private String estadoCivil;

        public String EstadoCivil
        {
            get { return estadoCivil; }
            set { estadoCivil = value; }
        }

    }

    
}
